install_config/aggregate_logging.adoc

```diff
@@ -425,14 +446,16 @@ After deployment, the deployer can be removed:
 $ oc delete sa/logging-deployer secret/logging-deployer
 ----
 
-To remove everything generated without having to destroy the project:
+As well as everything generated during the deployment that will not destroy the project:
 
```

Comment:
Is it weird to continue a sentence here after a command? Could this be more like "You can also remove everything..."?